specification/schemaregistry/data-plane/Microsoft.EventHub/stable/2022-10/schemaregistry.json

```diff
@@ -0,0 +1,614 @@
+              "Schema-Version": {
+                "type": "integer",
+                "description": "Version of the returned schema."
+              }
+            },
+            "schema": {
+              "description": "Binary representation of the registered schema.",
+              "type": "file"
+            }
+          },
+          "default": {
+            "description": "An error response received from the Azure Schema Registry service.",
+            "headers": {
+              "x-ms-error-code": {
+                "type": "string",
+                "description": "Error code for specific error that occurred."
+              }
+            },
+            "schema": {
+              "$ref": "#/definitions/Error"
+            }
+          }
+        }
+      }
+    },
+    "/$schemaGroups/{groupName}/schemas/{schemaName}/versions": {
+      "get": {
+        "summary": "Get list schema versions.",
+        "description": "Gets the list of all versions of one schema.",
+        "x-ms-examples": {
+          "Get schema by ID": {
+            "$ref": "./examples/OperationSchema_GetSchemaVersions.json"
+          }
+        },
+        "operationId": "Schema_GetVersions",
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "name": "groupName",
+            "in": "path",
+            "description": "Schema group under which schema is registered.  Group's serialization type should match the serialization type specified in the request.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "$ref": "#/parameters/SchemaName"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/SchemaVersions"
+            }
+          },
+          "default": {
+            "description": "An error response received from the Azure Schema Registry service.",
+            "headers": {
+              "x-ms-error-code": {
+                "type": "string",
+                "description": "Error code for specific error that occurred."
+              }
+            },
+            "schema": {
+              "$ref": "#/definitions/Error"
+            }
+          }
+        }
+      }
+    },
+    "/$schemaGroups/{groupName}/schemas/{schemaName}/versions/{schemaVersion}": {
+      "get": {
+        "summary": "Get specific schema versions.",
+        "description": "Gets one specific version of one schema.",
+        "x-ms-examples": {
+          "Get schema by ID": {
+            "$ref": "./examples/OperationSchema_GetByVersion.json"
+          }
+        },
+        "operationId": "Schema_GetSchemaVersion",
+        "produces": [
+          "application/json; serialization=Avro",
+          "application/schema+json",
+          "application/octet-stream"
```

Comment:
What do you think about using text/plain for custom schemas? Since we've defined SchemaDefinition as a string in the libraries, we will be forcing users to specify a string value for custom schemas as well.

Reply:
@hmlam Since the schema comes in as a string from the client side, do you think we should move forward with changing out content type to "text/plain" for custom schemas?

Reply:
After some follow up, we'll go with `text/plain; charset=utf-8` for the request and response headers for custom schemas.